### ISSUE TITLE
Null Safety

### ISSUE THREAD

**Issue by jonbhanson** [FooStudio/tinycolor#15]
Thanks for the great package!

Do you plan to make a null safe update of the package?

**Comment by georgeherby** [comment]
Fixed by #13

**Comment by WieFel** [comment]
Any news on this?
I'd need null safety on this package to be able to migrate my app...

**Comment by AjayShanker-geek** [comment]
Hi, any update?

**Comment by calvintam236** [comment]
I think it is a good time to ask if @Skquark @mendieta can merge PR #13 and release new version on pub.dev.

**Comment by WieFel** [comment]
@calvintam236 true. It seems there are not many updates happening on this package anymore.

If @Skquark @mendieta don't have time, someone could join the maintainers to be able to merge PRs and release new versions.
I would be interested for example...

**Comment by calvintam236** [comment]
> @calvintam236 true. It seems there are not many updates happening on this package anymore.
> 
> If @Skquark @mendieta don't have time, someone could join the maintainers to be able to merge PRs and release new versions.
> I would be interested for example...

If they are not going to response soon, I think it is a good time to start a community fork `tinycolor2`.

**Comment by WieFel** [comment]
@calvintam236 exactly

**Comment by TNorbury** [comment]
Should it be called tinycolor 2 2 or tinycolor 3? 🤔

**Comment by calvintam236** [comment]
> Should it be called tinycolor 2 2 or tinycolor 3? 🤔

Current version is v1.0.3 so `tinycolor2` for v2 make sense.

`tinycolor2` is also the original package name, which is a perfect match.

Node.js side: https://www.npmjs.com/package/tinycolor2

**Comment by WieFel** [comment]
@calvintam236 should I go ahead and fork the repo? I can add you as co-maintainer...

**Comment by calvintam236** [comment]
> @calvintam236 should I go ahead and fork the repo? I can add you as co-maintainer...

Sure. That works for me. You can do it under an organization since it is community based.

**Comment by WieFel** [comment]
@calvintam236 I think I'd first fork it on my personal account and then in a second step propose it e.g. to [fluttercommunity organization](https://github.com/fluttercommunity). Except you already have some organization!?

**Comment by calvintam236** [comment]
> @calvintam236 I think I'd first fork it on my personal account and then in a second step propose it e.g. to [fluttercommunity organization](https://github.com/fluttercommunity). Except you already have some organization!?

https://github.com/fluttercommunity/community#note-to-packages-owners

Let’s create an organization for this project, maybe `tinycommunity`? 🤣
This way, both of us can publish to pub.dev as organization admin. After we got everything clean up and updated, then we can go for `fluttercommunity` or remains as standalone. That decision depends on the community.

**Comment by calvintam236** [comment]
Hey all, [community-forked repo](https://github.com/TinyCommunity/tinycolor2) is live now. Please feel free to contribute PRs while we are working to update the code and publish to pub.dev.

**Comment by manukumarsb7** [comment]
Hi, Any news on this?

**Comment by WieFel** [comment]
> Hey all, [community-forked repo](https://github.com/TinyCommunity/tinycolor2) is live now. Please feel free to contribute PRs while we are working to update the code and publish to pub.dev.

@manukumarsb7 check this

**Comment by rkoshti** [comment]
Hi @WieFel @calvintam236 When you guys are planning to publish null safety update on pub.dev ?

**Comment by WieFel** [comment]
@rkoshti it is already released for some time. Be aware that therefore you have to use our new package `tinycolor2`:
https://pub.dev/packages/tinycolor2
